drivers/lenel/open_access.cr

```diff
@@ -34,7 +34,7 @@ class Lenel::OpenAccess < PlaceOS::Driver
   end
 
   private getter default_timezone : String do
-    setting?(String, :timezone) || "Asia/Dubai"
+    setting?(String, :timezone) || "UTC"
   end
```

Comment:
```suggestion
  private getter default_timezone : Time::Location do
    tz = setting?(String, :timezone) || "UTC"
    Time::Location.load tz
  end
```

Reply:
thanks yeah that totally makes sense, i'll add it